### DISCUSSION TITLE
Better option for key shortcut for Interactive Mode

### DISCUSSION THREAD

**Discussion by rampatra** [Softal-io/presentify-community#13] [General]
I received feedback to change either the behavior of the Fn key shortcut such that you only need to press it once instead of press and hold or to provide another key shortcut apart from caps lock for this.

So, I would like to ask the community about this and implement the best solution possible in the next update.

Thanks,
Ram

**Comment by archerbird** [comment]
Yes please, this would help me quite a lot. I would like to see custom binding, and/or at least some boolean to flip between "tap-and-hold" to toggle vs "tap to toggle"
